core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message.groovy

```diff
@@ -29,7 +29,7 @@ dl {
     div(class: "alert alert-info") {
         a(name: "resource-root-url")
         form(method: "post", action: "${rootURL}/${my.url}/act") {
-            f.submit(name: 'redirect', value: _("Go to resource root URL configuration"))
+            f.submit(name: 'redirect', value: _("Configure resource root URL"))
```

Comment:
Impacts localizations if any

Reply:
How about just improve it in the English properties?

Reply:
@oleg-nenashev: There appear to be 0 in tree localizations.
@LinuxSuRen: I'm trying to change the meaning, so if someone actually did translate this message, I'd like them to think about the new key and try translating the new spirit.